### PR TITLE
GH-854: support section

### PR DESCRIPTION
**Pull request by Midnite1212**

### CODE REVIEW

File: ui/src/components/ripple-out/RippleOutTextStyle.js
Comment (ksuhartono97) [inline ui/src/components/ripple-out/RippleOutTextStyle.js:1]: wait this means all the other pages need to align and use this, i guess it's ok for now but we need to ensure everyone uses these constants (unless custom needs)

File: ui/src/components/ripple-out/RippleOutSupportContainer.js [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+              <Text
+                fontSize={subHeaderFontSize}
+                textStyle="darker_grotesque_black"
+                lineHeight="0.9em"
+              >
+                WAYS TO SUPPORT
+              </Text>
+              <Text fontSize={bodyFontSize}>{supportWaysText}</Text>
+            </Flex>
+            {/* Beginning of tabs */}
```
Comment (ksuhartono97) [inline ui/src/components/ripple-out/RippleOutSupportContainer.js:94]: image
ipad view overflows

File: ui/src/components/ripple-out/RippleOutSupportGiveContainer.js
Comment (ksuhartono97) [inline ui/src/components/ripple-out/RippleOutSupportGiveContainer.js:1]: is this file supposed to be here. i think u deleted it but it seems like it got readded after u rebased

File: ui/src/components/ripple-out/support-sections/RippleOutGiveSection.js [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+      <Flex>
+        <VStack>
+          <Text fontSize={bodyFontSize} fontWeight={500}>
+            Here’s an illustration of how a small act of giving over time can go
+            a long way to contribute to the{' '}
+            <Box as="span" fontWeight={700}>
+              Ripple Out Campaign
+            </Box>
+          </Text>
+          <Image
```
Comment (ksuhartono97) [inline ui/src/components/ripple-out/support-sections/RippleOutGiveSection.js:103]: Need a swipe interaction for this according to design. atm the only way for people to switch between slides is to click dots which is not very intuitive

File: ui/src/components/ripple-out/support-sections/RippleOutPraySection.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+        <VStack spacing={8} display="flex" alignItems="flex-start">
+          <Text
+            fontSize={['1.4em', '2.4em']}
+            textStyle="darker_grotesque_black"
+            lineHeight="0.9em"
+          >
+            PRAY FOR THE CAMPAIGN
+          </Text>
+
+          <VStack w="100%" display="flex" alignItems="flex-start" spacing={6}>
```
Comment (ksuhartono97) [inline ui/src/components/ripple-out/support-sections/RippleOutPraySection.js:56]: Need a swipe interaction for this according to design. atm the only way for people to switch between slides is to click dots which is not very intuitive

File: ui/src/components/ripple-out/support-sections/RippleOutPraySection.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+          borderRadius="30px"
+          display={['flex', 'none']}
+          onClick={() => setSlider(i)}
+          cursor="pointer"
+        />
+      </>
+    );
+  }
+  return (
+    <>
```
Comment (ksuhartono97) [inline ui/src/components/ripple-out/support-sections/RippleOutPraySection.js:33]: the <></> is unnecessary since it's a 1 element item

**Review by ksuhartono97 — CHANGES_REQUESTED** [review]

File: ui/src/components/ripple-out/RippleOutTextStyle.js
Comment (Midnite1212) [reply ui/src/components/ripple-out/RippleOutTextStyle.js:1]: all of the pages should have alrd uses this, I just refactored the styles into one file in each page in my prev restructure pr.

**Review by ksuhartono97 — APPROVED** [review]

**Pull request merged**